templates/login.html

```diff
@@ -2,24 +2,26 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <title>Microsoft Identity Python Web App: Login</title>
 </head>
 <body>
     <h1>Microsoft Identity Python Web App</h1>
 
     {% if user_code %}
     <ol>
-    <li>To sign in, type <b>{{ user_code }}</b> into
-      <a href='{{ auth_uri }}' target=_blank>{{ auth_uri }}</a>
-      to authenticate.
-    </li>
-    <li>And then <a href="{{ url_for('auth_response') }}">proceed</a>.</li>
+      <li>To sign in, type <b>{{ user_code }}</b> into
+        <a href='{{ auth_uri }}' target=_blank>{{ auth_uri }}</a>
+        to authenticate.
+      </li>
+      <li>And then <a href="{{ url_for('auth_response') }}">proceed</a>.</li>
     </ol>
+    {% elif error %}
```

Comment:
Did we accidentally add this line?
```suggestion
```

Reply:
Huh. Maybe Copilot did it?? Ohhh I think it did, as it makes a similar suggestion as soon as I delete it. Sorry, must have accidentally accepted it! Sneaky copilot.